src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
 
 [id="configuration"]
 == Configuration
 
-Che CLI allows a wide range of config changes to setup port, hostname, oAuth, Docker, git, and solve networking issues. See: link:docker-config.html[Che configuration on Docker].
+Che CLI allows a wide range of configuration changes to setup the port, hostname, oAuth, Docker, git, and to solve networking issues. For details, see link:docker-config.html[Che configuration on Docker].
 
 [id="run-without-cli"]
-== Run Without CLI
+== Running Che without the CLI
+
+You can run the Che server directly by launching a Docker image. This approach bypasses the CLI that has additional utilities to simplify administration and operation. The `eclipse/che-server` Docker image is appropriate for running Che within clusters, orchestrators, or by third-party tools with automation.
 
-You can run the Che server directly by launching a Docker image. This approach bypasses the CLI, which has additional utilities to simplify administration and operation. The `eclipse/che-server` Docker image is appropriate for running Che within clusters, orchestrators, or by third party tools with automation.
+To run Che directly by launching a Docker image, take the following steps:
 
+. Run the latest released version of Che.
+. Replace <LOCAL_PATH> with any host folder. Che places the backup files in this folder - configurable properties, workspaces, lib, storage.
++
 ----
-# Run the latest released version of Che
-# Replace <LOCAL_PATH> with any host folder
-# Che will place backup files there - configurable properties, workspaces, lib, storage
 docker run -p 8080:8080 \
            --name che \
            --rm \
            -v /var/run/docker.sock:/var/run/docker.sock \
            -v <LOCAL_PATH>:/data \
            eclipse/che-server:6.0.0
+----
 
-# To run the nightly version of Che, replace eclipse/che-server:5.0.0-latest with
-eclipse/che-server:nightly
+Following are some useful commands:
 
-# To run a specific tagged version of Che, replace eclipse/che-server:5.0.0-latest with
-eclipse/che-server:<version>
+* To run the nightly version of Che, replace `eclipse/che-server:5.0.0-latest` with `eclipse/che-server:nightly`.
 
-# Stop the container running Che
-docker stop che
+* To run a specific tagged version of Che, replace `eclipse/che-server:5.0.0-latest` with `eclipse/che-server:<_version_>`.
 
-# Restart the container running Che and restart the Che server
-docker restart che
+* To Stop the container running Che, use the `docker stop che` command.
+
+* To restart the container running Che and restart the Che server, use the `docker restart che` command.
 
-# Upgrade to a newer version
+* To upgrade to a newer version, use the following commands in order:
++
+----
 docker pull eclipse/che-server:6.0.0-latest
```

Comment:
using a hard-coded version seems like a mistake; current version is already what? 6.1.12?

Reply:
I changed that to 
"docker pull eclipse/che-server:<_latest-version_>
docker restart che"

Also, removed the hard-coded versions in the following commands:
SELinux:
docker run -p 8080:8080 \
           --name che \
           -v /var/run/docker.sock:/var/run/docker.sock \
           -v <LOCAL_PATH>:/data:Z \
           --security-opt label:disable \
           eclipse/che-server:<_latest-version_>

Ports:
docker run -p 9000:9500 \
           --name che \
           -e CHE_PORT=9500 \
           -v /var/run/docker.sock:/var/run/docker.sock \
           -v <LOCAL_PATH>:/data \
           eclipse/che-server:<_latest-version_>

Better?

Reply:
+1